docker/playground-ams-entrypoint.sh

```diff
@@ -1,3 +1,14 @@
 #!/bin/bash
-sleep 20
+
+SQL_RESULT=$(mysql -h mysql -P 3306 --database=arctic --user=root --password=mysql --execute="show tables like 'ddl_record';" -s -N)
+
+while [[ "${SQL_RESULT}" != "ddl_record" ]]
+do
+	echo "Wait for table initialization to complete..."
+	sleep 1
+	SQL_RESULT=$(mysql -h mysql -P 3306 --database=arctic --user=root --password=mysql --execute="show tables like 'ddl_record';" -s -N)
```

Comment:
the duplicated code can be simplified by do ... while

Reply:
oh sorry, let's init the SQL_RESULT w/ a static string to avoid code duplication.

Reply:
i don't know how to do this, i add `` to enclose the MySQL COMMAND, but it did not work